package.json

```diff
@@ -215,6 +215,6 @@
   },
   "lint-staged": {
     "*.{js,ts}": ["biome check --apply"],
-    "*.json": ["biome format --write"]
+    "*.json": ["biome format --write --no-errors-on-unmatched"]
```

Comment:
Oh, this is a nice catch! I've been wondering why I've been getting that error sometimes 😄